### ISSUE TITLE
Reactive plots disappear after browser window refresh

### ISSUE THREAD

**Issue by AnatolyVuets** [rstudio/flexdashboard#164]
Hello, flexdashboard team! 
I would appreciate any help on the following issue: 
Reactive plots disappear when I refresh my dashboard page in browser. 
I'm using the most recent development versions of flexdashboard, shiny and plotly.
RStudio v 1.1.383; R v 3.4.3

Code example:
```{r}
## some data preparations 

 renderPlotly({
selectedData() %>%
  plot_ly(x = ~funnel_ymd, y = ~value, color = ~variable, colors = "Dark2",
          yaxis = ~paste0("y", id)) %>%
  add_bars() %>%
  subplot(nrows = 14, shareX = TRUE)
})
```

However, analogous non-reactive plot does not suffer from this issue:

```{r}
## some data preparations 
constData %>%
  plot_ly(x = ~funnel_ymd, y = ~value, color = ~variable, colors = "Dark2",
          yaxis = ~paste0("y", id)) %>%
  add_bars() %>%
  subplot(nrows = 14, shareX = TRUE)

```

**Comment by dwall1** [comment]
Did you ever find a way to resolve this issue? I'm experiencing something similar withmy flexdashboard app

**Comment by AnatolyVuets** [comment]
dwall1, unfortunately I did not manage to solve this problem and ended up using Shiny Dashboard which if more flexible and rich in options. I would recommend you to try it

**Comment by dwall1** [comment]
Thanks for letting me know

**Comment by Ljupch0** [comment]
I'm having the same issue on all of my flexdashboard runtime:shiny apps. Any update?

**Comment by Ljupch0** [comment]
It seems that shinydashboard is the way to go.

**Comment by dvictori** [comment]
I'm also facing this issue in a flexdashboard that connects to a Postgres database. Has anyone solved this?

**Comment by dvictori** [comment]
I've found out what is causing the problem. In my original flexdashborad, when I added `runtime: shiny`, the text was indented at the same level of the `output` block. So when I reloaded the flexdashboard - even in RStudio viewer - the rendered outputs would stop working. 

When I removed the indentation from `runtime:shiny`, things got back to normal. So for anyone having this issue, check your indentation on the YAML at the top of the document.
